pkg/receive/multitsdb.go

```diff
@@ -388,7 +388,7 @@ func (s *ReadyStorage) Querier(ctx context.Context, mint, maxt int64) (storage.Q
 // Appender implements the Storage interface.
 func (s *ReadyStorage) Appender() (storage.Appender, error) {
 	if x := s.get(); x != nil {
-		return x.Appender()
+		return x.Appender(context.TODO())
```

Comment:
@bwplotka What do you think about propagating the context to `ReadyStorage.Appender()`? I've looked at parent callers and looks an easy thing to do: `Handler.fanoutForward()` > `Writer.Write()` > this function.

Reply:
yes I think passing this makes a lot of sense

Reply:
Thanks, addressed.